src/Atmos/Model/radiation.jl

```diff
@@ -1,5 +1,5 @@
 using CLIMA.PlanetParameters
-export NoRadiation, StevensRadiation
+export NoRadiation, DYCOMSRadiation
```

Comment:
We should move all of DYCOMSRadiation to `experiments/LES/dycoms.jl` in PR #597 when ready,

Reply:
Opened #616 as a reminder, we can do this in #597